sympy/groups/matrix_groups.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import print_function
+from sympy.core import Basic
+from sympy import Matrix
+from sympy.ntheory.primetest import isprime
+from math import sin, cos, pi
```

Comment:
Why are you using functions from math? In general, SymPy code should use symbolic functions from sympy.